### PR TITLE
SparkLend Ethereum - Increase Capacity of weETH - Governance Poll 2024-07-01

### PR DESCRIPTION
**Pull request by blimpa**
Originating forum post: https://forum.makerdao.com/t/jun-27-2024-proposed-changes-to-spark-for-upcoming-spell/24552

### CODE REVIEW

File: governance/polls/SparkLend Ethereum - Increase Capacity of weETH - July 1, 2024.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+**This is a binary vote.**
+
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll allows MakerDAO Governance to signal their support or opposition to increasing the capacity of weETH (Etherfi weETH) on mainnet with the following parameters:
```
Comment (VoteWizard) [inline governance/polls/SparkLend Ethereum - Increase Capacity of weETH - July 1, 2024.md:30]:
```suggestion
This poll allows MakerDAO Governance to signal their support or opposition to increasing the capacity of [weETH (Etherfi weETH)](https://app.spark.fi/market-details/1/0xCd5fE23C85820F7B72D0926FC9b05b43E359b7ee) on mainnet with the following parameters:
```

File: governance/polls/SparkLend Ethereum - Increase Capacity of weETH - July 1, 2024.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+**This is a binary vote.**
+
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll allows MakerDAO Governance to signal their support or opposition to increasing the capacity of [weETH (Etherfi weETH)](https://app.spark.fi/market-details/1/0xCd5fE23C85820F7B72D0926FC9b05b43E359b7ee) on mainnet with the following parameters:
```
Comment (VoteWizard) [inline governance/polls/SparkLend Ethereum - Increase Capacity of weETH - July 1, 2024.md:30]:
```suggestion
This poll allows MakerDAO Governance to signal their support or opposition to increasing the capacity of [weETH (Etherfi weETH)](https://app.spark.fi/market-details/1/0xCd5fE23C85820F7B72D0926FC9b05b43E359b7ee) on mainnet with the following parameter changes:
```

File: governance/polls/SparkLend Ethereum - Increase Capacity of weETH - July 1, 2024.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+- weETH Supply Cap Max: **200,000 weETH**
```
Comment (VoteWizard) [inline governance/polls/SparkLend Ethereum - Increase Capacity of weETH - July 1, 2024.md:32]:
```suggestion
- Increase weETH Supply Cap Max by 150,000 weETH, from 50,000 weETH to **200,000 weETH**
```

File: governance/polls/SparkLend Ethereum - Increase Capacity of weETH - July 1, 2024.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+- weETH Isolation Mode Debt Ceiling: **200 million DAI**
```
Comment (VoteWizard) [inline governance/polls/SparkLend Ethereum - Increase Capacity of weETH - July 1, 2024.md:33]:
```suggestion
- Increase weETH Isolation Mode Debt Ceiling by  150 million DAI, from 50 million DAI to **200 million DAI**
```

File: governance/polls/SparkLend Ethereum - Increase Capacity of weETH - July 1, 2024.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll allows MakerDAO Governance to signal their support or opposition to increasing the capacity of [weETH (Etherfi weETH)](https://app.spark.fi/market-details/1/0xCd5fE23C85820F7B72D0926FC9b05b43E359b7ee) on mainnet with the following parameter changes:
+
+- Increase weETH Supply Cap Max by 150,000 weETH, from 50,000 weETH to **200,000 weETH**
+- Increase weETH Isolation Mode Debt Ceiling by 150 million DAI, from 50 million DAI to **200 million DAI**
```
Comment (VoteWizard) [inline governance/polls/SparkLend Ethereum - Increase Capacity of weETH - July 1, 2024.md:33]:
```suggestion
- Increase weETH [Isolation Mode](https://docs.spark.fi/defi-infrastructure/sparklend/spark-lend-features) Debt Ceiling by 150 million DAI, from 50 million DAI to **200 million DAI**
```

File: governance/polls/SparkLend Ethereum - Increase Capacity of weETH - July 1, 2024.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
+---
+title: SparkLend Ethereum - Increase Capacity of weETH - July 1, 2024
+summary: Increase capacity of WeETH (Etherfi weETH) on SparkLend Ethereum.
```
Comment (0xldr) [inline governance/polls/SparkLend Ethereum - Increase Capacity of weETH - July 1, 2024.md:3]:
```suggestion
summary: Increase capacity of weETH (Etherfi weETH) on SparkLend Ethereum.
```

**Review by 0xldr — CHANGES_REQUESTED** [review]
Only one small change otherwise good to merge

**Pull request merged**